i18n/zh/docusaurus-theme-classic/footer.json

```diff
@@ -1,6 +1,6 @@
 {
   "copyright": {
-    "message": "2024 © KubeEdge Project Authors. All rights reserved.",
+    "message": "2025 © KubeEdge Project Authors. All rights reserved.",
```

Comment:
IMO https://github.com/kubeedge/website/issues/597 requires avoiding this update with auto-generated year, so statically putting the year here is not really what https://github.com/kubeedge/website/issues/597 wants?